example.env

```diff
@@ -7,3 +7,7 @@ JWT_SECRET=
 BCRYPT_SALT_ROUNDS=
 REDIS_PASSWORD=
 REDIS_URL=
+AWS_S3_BUCKET=
+AWS_S3_ACCESS_KEY=
+AWS_S3_KEY_SECRET=
```

Comment:
for these 2 environment variables I would use default names used by AWS SDK, that is `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`: https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/loading-node-credentials-environment.html